packages/vite/src/node/plugins/worker.ts

```diff
@@ -6,13 +6,19 @@ import type Rollup from 'rollup'
 import { ENV_PUBLIC_PATH } from '../constants'
 import path from 'path'
 import { onRollupWarning } from '../build'
+import type { TransformPluginContext } from 'rollup'
 
 const WorkerFileId = 'worker_file'
 
+export interface BundleWorkerEntryOutput {
+  code: string
+  sourcemap: Rollup.SourceMap | undefined
+}
+
 export async function bundleWorkerEntry(
```

Comment:
`bundleWorkerEntry` will always use with `emitSourcemapForWorkerEntry`. In the laster version just diff with params `query`, may be `emitSourcemapForWorkerEntry` can exec in `bundleWorkerEntry`?